### PR TITLE
Add Prettier and format code

### PR DESCRIPTION
**Pull request by jeremymeng**
Better formatting makes working with code more pleasant.

- Enforce tslint-eslint-rules ter-indent `SwitchCase` indentation level of 1 to match prettier behavior.

### CODE REVIEW

File: .scripts/checkEverything.ts [hunk 1]
```diff
@@ -4,6 +4,6 @@ import { checkConstantsVersion } from "./checkConstantsVersion";
 checkEverything({
   additionalChecks: {
     name: "Constants.ts Version",
-    check: checkConstantsVersion
-  }
+    check: checkConstantsVersion,
```
Comment (jeremymeng) [inline .scripts/checkEverything.ts:7]: in prettier v2.0, trailingComma:  "es5" is now default. It makes adding new entries for multi-line lists easier.  https://prettier.io/docs/en/options.html#trailing-commas

**Comment by ramya-rao-a** [comment]
I get why the Check_Everything step is failing in CI, but here the test step is also failing. Looks like the webpack step is failing there

> ERROR: /home/vsts/work/1/s/lib/httpPipelineLogger.ts:43:1 - Expected indentation of 6 spaces but found 8.
ERROR: /home/vsts/work/1/s/lib/httpPipelineLogger.ts:44:1 - Expected indentation of 6 spaces but found 8.
ERROR: /home/vsts/work/1/s/lib/httpPipelineLogger.ts:46:1 - Expected indentation of 4 spaces but found 6.
ERROR: /home/vsts/work/1/s/lib/httpPipelineLogger.ts:47:1 - Expected indentation of 6 spaces but found 8.

**Review by xirzec — APPROVED** [review]
Not reviewing this one very thoroughly if it's just the result of calling format

**Comment by jeremymeng** [comment]
> I get why the Check_Everything step is failing in CI, but here the test step is also failing. Looks like the webpack step is failing there

Very likely broken by the formatting changes. Looking into it.

**Comment by jeremymeng** [comment]
- Prettier broke lines but the // tslint:disable-next-line comments remains in
the old position. They should be before the applicable lines.

- Enforce `SwitchCase` indentation level of 1 to match prettier behavior.

**Comment by jeremymeng** [comment]
@ramya-rao-a CI now only fails due to version.

**Review by ramya-rao-a — APPROVED** [review]

**Pull request merged**